src/core/execute.h

```diff
@@ -379,7 +379,7 @@ typedef enum ExecFlags {
         EXEC_NSS_DYNAMIC_BYPASS    = 1 << 5, /* Set the SYSTEMD_NSS_DYNAMIC_BYPASS environment variable, to disable nss-systemd blocking on PID 1, for use by dbus-daemon */
         EXEC_CGROUP_DELEGATE       = 1 << 6,
         EXEC_IS_CONTROL            = 1 << 7,
-        EXEC_CONTROL_CGROUP        = 1 << 8, /* Place the process not in the indicated cgroup but in a subcgroup '/.control', but only EXEC_CGROUP_DELEGATE and EXEC_IS_CONTROL is set, too */
+        EXEC_CONTROL_CGROUP        = 1 << 8, /* Place the process not in the indicated cgroup but in a subcgroup, but only when EXEC_CGROUP_DELEGATE is set, too */
```

Comment:
it's a bit of a misnomer when you apply this to the "main" process too. maybe add a separate flag for that, i.e. EXEC_MAIN_CGROUP or so?

Reply:
Good catch. I've flipped the flag for easier use.